.github/workflows/java-build.yml

```diff
@@ -0,0 +1,28 @@
+name: Java (build)
+
+on:
+  push:
+    paths:
+      - 'visual-java/**'
+  pull_request:
+    paths:
+      - 'visual-java/**'
+
+defaults:
+  run:
+    working-directory: visual-java
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '11'
```

Comment:
Shouldn't we compile it with Java8 since right now we expect to support starting version 8 ?

Reply:
Yes 8 makes more sense, will update